### PR TITLE
Add Cembra MoneyBank

### LINKED ISSUE #5037: Add Cembra Money Bank
- [x] I have searched both open and closed [issues](https://github.com/2factorauth/twofactorauth/issues). The issue I'm creating is not a duplicate of an existing issue.
- [x] I have searched both open and closed [pull requests](https://github.com/2factorauth/twofactorauth/pulls). The issue I'm creating is not a duplicate of an existing pull request.

### Information about the site to be added that supports 2FA: ###

* Site name - `Cembra Money Bank`

* Site URL - `https://www.cembra.ch/en`

* Does the site support 2FA? - `Yes`

* Documentation URL - ``

### Information about what methods of 2FA the site supports: ###

* Software:
  - [ ] TOTP (RFC-6238, Google Authenticator)
  - [ ] Proprietary (Authy, Duo, etc.)

* Hardware:
  - [ ] FIDO2/U2F security keys
  - [ ] Other

- [ ] Phone calls
- [x] SMS tokens
- [ ] Email tokens

### Information about the site's eligibility to be added to twofactorauth.org: ###

- [x] I have checked that the site meets twofactorauth.org's [contributing guidelines and site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md).
- [x] The site is ranked within the top 200,000 sites on [Alexa](https://www.alexa.com/siteinfo/) or has a substantial social media following.
- [x] The site does not contain or promote content that violates twofactorauth.org's [excluded categories and websites guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md), including pornographic, discriminatory, or unlawful content.

### PR DESCRIPTION
**Pull request by christianbumann**
Resolves #5037

### CODE REVIEW

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
We currently don't support listing a site in multiple categories.
Please choose one and remove the site listing in the other category.

File: _data/banking.yml [hunk 1]
```diff
@@ -319,6 +319,12 @@ websites:
     facebook: CDCFederalCreditUnion
     img: cdcfcu.png
 
+  - name: Cembra Money Bank
+    url: https://www.cembra.ch/en
+    tfa:
+      - sms
+    img: cembra-money-bank.png
+
```
Comment (Carlgo11) [inline _data/banking.yml:330]:
```suggestion
    regions:
    - ch

```

File: _data/banking.yml [hunk 2]
```diff
@@ -319,6 +319,12 @@ websites:
     facebook: CDCFederalCreditUnion
     img: cdcfcu.png
 
+  - name: Cembra Money Bank
+    url: https://www.cembra.ch/en
```
Comment (Carlgo11) [inline _data/banking.yml]:
```suggestion
    url: https://www.cembra.ch/
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Thanks for the pull request!

I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
screenshot

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
The image provided is unnecessarily large. Here's a compressed version of their favicon:
icon

**Comment by phallobst** [comment]
There is a video (in German) that shows how the login procedure works: https://faq.cembra.ch/de/videos/login-mit-mtan/

Maybe this sufficient for the doc link.

**Comment by christianbumann** [comment]
I will look tomorrow on the open comments/recommendations

**Comment by christianbumann** [comment]
> 
> 
> There is a video (in German) that shows how the login procedure works: https://faq.cembra.ch/de/videos/login-mit-mtan/
> 
> Maybe this sufficient for the doc link.

I added the link to the video, as it shows the whole login process. Even it's in German, imho its better than only screenshots - and without understanding German the video should be self explained.

**Comment by christianbumann** [comment]
> 
> 
> The image provided is unnecessarily large. Here's a compressed version of their favicon:
> icon

I did a resizing myself and reduced the size 25%. I hope this is enough.

File: _data/banking.yml (hunk 1 above)
Comment (christianbumann) [reply _data/banking.yml:330]: Region added

**Comment by christianbumann** [comment]
> 
> 
> Thanks for the pull request!
> 
> I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?
> 
> If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
> **Remember to blur or crop out any personal information linked to you.**
> 
> You can attach the screenshots here in a comment, like this:
> screenshot

I added a link to a video. Its in German, but imho it is better than screenshots and the main audience speaks German, French or Italian. Even without sound, the video shows the login complete process understandable.

File: _data/banking.yml [hunk 3]
```diff
@@ -319,6 +319,15 @@ websites:
...
     img: cdcfcu.png
 
+  - name: Cembra Money Bank
+    url: https://www.cembra.ch/
+    tfa:
+      - sms
+    img: cembra-money-bank.png
+    regions:
+      - ch
+    doc: https://faq.cembra.ch/de/videos/login-mit-mtan/
```
Comment (phallobst) [inline _data/banking.yml]: Although it's not strictly necessary, we try to keep the lines for an entry in a certain order, so this is just a bit of shuffling around:
```suggestion
    img: cembra-money-bank.png
    tfa:
      - sms
    doc: https://faq.cembra.ch/de/videos/login-mit-mtan/
    regions:
      - ch
```

Comment (christianbumann) [reply _data/banking.yml]: @phallobst order changed

Comment (phallobst) [reply _data/banking.yml]: You could have just used "Commit suggestion" to apply the suggested changes, no need to do it manually.

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**